sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -122,3 +122,36 @@ ALTER TABLE `clinical_rules` ADD COLUMN `bibliographic_citation` VARCHAR(255) NO
 ALTER TABLE `clinical_rules` ADD COLUMN `linked_referential_cds` VARCHAR(50) NOT NULL DEFAULT '';
 #EndIf
 
+#IfMissingColumn clinical_rules amc_2015_flag
+ALTER TABLE `clinical_rules` ADD `amc_2015_flag` TINYINT(1) NULL DEFAULT NULL
+    COMMENT '2015 Automated Measure Calculation flag for (unable to customize per patient)'
+    AFTER `amc_2014_stage2_flag`;
```

Comment:
would avoid using AFTER since it's just a potential break if for some reason the column no longer exists (should never be case, but one never knows :) )